tfhe/src/core_crypto/algorithms/test/lwe_programmable_bootstrapping.rs

```diff
@@ -251,8 +247,6 @@ fn lwe_encrypt_pbs_f128_decrypt_custom_mod<
 
         convert_standard_lwe_bootstrap_key_to_fourier_128(&bsk, &mut fbsk);
 
-        drop(bsk);
```

Comment:
why remove this one ? it ensures the key is dropped here, is it a lint from clippy ?